src/components/shared/avatar-grid-box/AvatarGridBox.jsx

```diff
@@ -2,12 +2,19 @@ import PropTypes from "prop-types";
 import { Box } from "@mui/material";
 import { CustomAvatar } from "components";
 
-export const AvatarGridBox = ({ children, avatarSrc, username, className }) => {
+export const AvatarGridBox = ({
+  children,
+  username,
+  avatarSrc,
+  className,
+  handleClick,
+}) => {
   return (
     <Box
       gap={2}
       display="grid"
       className={className}
+      onClick={handleClick}
```

Comment:
handler name can be more descriptive

Reply:
Thanks for suggesting @bhavyaj12 
I was trying to keep a generic name but will look once more if I can name something different and will add the changes in the future PR.